### PR TITLE
Add start menu links for zip installers

### PR DESCRIPTION
**Pull request by mimi89999**

### CODE REVIEW

File: pkg/justinstall/registry.go [hunk 1]
```diff
@@ -283,7 +285,25 @@ func (e *RegistryEntry) install(path string) error {
...
+			return err
+		}
+
+		if shortcuts, prs := e.Installer.options()["shortcuts"]; prs {
+			for _, shortcut := range shortcuts.([]interface{}) {
+				shortcutName := expandString(shortcut.(map[string]interface{})["name"].(string), nil)
+				shortcutTarget := expandString(os.ExpandEnv(shortcut.(map[string]interface{})["target"].(string)), nil)
+				shortcutLocation := filepath.Join(startMenu, shortcutName + ".lnk")
+
+				log.Println("Creating shortcut to %s in %s", shortcutTarget, shortcutLocation)
```
Comment (lvillani) [inline pkg/justinstall/registry.go]:
```suggestion
				log.Println("Creating shortcut to", shortcutTarget, "in", shortcutLocation)
```

Comment (mimi89999) [reply pkg/justinstall/registry.go]: Done

**Comment by mimi89999** [comment]
@lvillani 
> I'm a bit worried about `gotopkg/mslnk` which is a complete re-implementation of the `.lnk` file format but it appears to work just fine for now.

Microsoft only provides libraries for C++, C# and VBScript. There are third party implementations in Bash, C, Java and Python 3. Re-implementing it in Go was the only option.

**Comment by geek1011** [comment]
The other option is to use OLE with go-ole to call CreateShortcut.

File: pkg/justinstall/registry.go (hunk 1 above)
Comment (geek1011) [reply pkg/justinstall/registry.go]: Also, for the future, note that you meant to use `Printf` with a `\n`, as `Println` doesn't support format arguments.

**Comment by mimi89999** [comment]
@lvillani When will you make a new release, so I can start adding links in the registry file?

**Comment by mimi89999** [comment]
@lvillani Any news about the possibility of making a new release?

**Comment by lvillani** [comment]
It will be released with just-install v4 "when it's ready". The change landed amidst a refactoring and will have to be back-ported if you have urgency to see this fixed in a point release.

**Pull request merged**